kart/lfs_commands/__init__.py

```diff
@@ -73,7 +76,8 @@ def pre_push(ctx, remote_name, remote_url, dry_run):
             # TODO - capture chunk progress and report our own overall progress
             subprocess.check_call(
                 ["git-lfs", "push", remote_name, "--object-id", *lfs_oids],
-                env=tool_environment,
+                env=tool_environment(),
+                cwd=repo.workdir_path,
```

Comment:
Hmm currently it's never null - it just falls back to the git-dir in a bare repo, so, it's the most obvious place from which you would run a git-lfs command. Will leave it as-is for now but it would need to be changed if we ever change the semantics of repo.workdir_path